packages/pinpin/README.md

```diff
@@ -0,0 +1,35 @@
+<h1 align="center">⁂<br/>web3.storage</h1>
+<p align="center">Pin pins on Piñata ✨</p>
+
+Backup all content pinned on our IPFS Cluster to Piñata's infra.
+
+It's a task queue! Fetch the oldest 600 PinRequests from the DB and pin them on Piñata, then delete the PinRequests from the db when successfully submitted.
+- At the time of writing the Fauna query to fetch PinRequests hits errors if a batch size other than 600 is used.
+- We can only upload to Piñata at 3req/s max, as they have rate-limiting in place.
+
+## Getting started
+
+Ensure you have all the dependencies, by running `npm i` in the parent project.
+
+To run this locally you will need the following in your `packages/cron/.env` file:
```

Comment:
Can we use `.env.local`?

Reply:
where does the `.env.local` pattern come from? we're not using the .env file for production, just as a convenience for developers... also notable https://github.com/motdotla/dotenv#should-i-have-multiple-env-files

Reply:
We already use the `.env.local` pattern in our codebase. The pattern was probably doing like nft.storage:

- https://github.com/web3-storage/web3.storage/blob/6cf8571c79505db4769a40bf42b36d484d315550/packages/api/webpack.config.js#L9
- https://github.com/web3-storage/web3.storage/blob/dfeffd6f4d529fe5765ee0ea12d0a813f2e951af/packages/db/tools/import-graphql-schema.js#L5

And we ignore it to be commited, while a .env file is whitelisted 
- https://github.com/web3-storage/web3.storage/blob/main/.gitignore#L26

Reply:
aye, i think my point was that we're not using a .env file in production, so we don't need the destinction between local and prod here. the env file is just to make it easier for a dev to set the env vars.

Reply:
My concern is that adding this can easily leak keys into github repo, given `.env` file is not in .gitginore and can be unintentionally committed quite easily. But this is a nit